_hooks/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Just a general comment: we use the Google style guidelines for most languages, so for this one, we'll want to adhere to the shell style guide:

https://google.github.io/styleguide/shell.xml